File: src/main/java/com/addressbook/dto/PersonDto.java
package com.addressbook.dto;

import com.addressbook.model.Person;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@ToString
@SuppressWarnings("unused")
public class PersonDto implements Serializable {

    private String id;
    private String orgId;
    private String firstName;
    private String lastName;
    private String resume;
    private String salary;

    public PersonDto(Person person) {
        this.id = person.getId();
        this.orgId = person.getOrgId();
        this.firstName = person.getFirstName();
        this.lastName = person.getLastName();
        this.resume = person.getResume();
        this.salary = String.valueOf(person.getSalary());
    }
}

File: addressbook-common/src/main/kotlin/com/addressbook/dto/PersonDto.kt
package com.addressbook.dto

import com.addressbook.model.Person
import java.io.Serializable

class PersonDto constructor(person: Person? = null) : Serializable {

    var id: String? = null
    var orgId: String? = null
    var firstName: String? = null
    var lastName: String? = null
    var resume: String? = null
    var salary: String? = null

    init {
        this.id = person?.id
        this.orgId = person?.orgId
        this.firstName = person?.firstName
        this.lastName = person?.lastName
        this.resume = person?.resume
        this.salary = person?.salary
    }
}
